README.md

```diff
@@ -131,4 +131,33 @@ websearch:
   # on your system. You can specify absolute path to an image file as well,
   # e.g. `/home/me/.projektor/google.png`
   icon: web-browser
+ui:
+  # use as a string. In a near future it will be proportinal to screen size (%)
+  width: "600"
+  height: "300"
 ```
+
+Projektor styles should be placed at `~/.projektor/styles.css`.
+
+Here's is de default style:
```

Comment:
This part should be spelled `Here is the default style:`

Reply:
It was a typo :man_facepalming: